packages/matrix/docker/synapse/templates/password_reset_success.html

```diff
@@ -0,0 +1,6 @@
+{% extends "_base.html" %}
+{% block title %}Password reset confirmation success{% endblock %}
+
+{% block content %}
+<p>Your email has now been validated, please return to Boxel forgot password screen to reset your password. You may now close this window.</p>
```

Comment:
I think It would be easier if you add a link to the password reset form here and urge the user to click it. Then you can remove the necessity to urge the user to go back the original window (what if the user closed it in the meanwhile?) and get rid of the "i have validated"  button mechanism